src/colorizer/Collection.ts

```diff
@@ -348,14 +387,14 @@ export default class Collection {
    * @returns a new Collection, where the only dataset is that of the provided `datasetUrl`.
    * The `url` field of the Collection will also be set to `null`.
    */
-  public static makeCollectionFromSingleDataset(datasetUrl: string): Collection {
+  public static makeCollectionFromSingleDataset(datasetUrl: string, options: CollectionLoadOptions = {}): Collection {
     // Add the default filename if the url is not a .JSON path.
     if (!isJson(datasetUrl)) {
-      datasetUrl = formatPath(datasetUrl) + "/" + DEFAULT_DATASET_FILENAME;
+      datasetUrl = Collection.joinPath(formatPath(datasetUrl), DEFAULT_DATASET_FILENAME);
```

Comment:
When loading from files, `datasetUrl` is an empty string. I added `joinPath` to handle that case, so it returns just the filename (`manifest.json`) instead of the filename with a slash in front of it (`/manifest.json`)